### PR TITLE
DOC: add line chart with sequence generator example

### PR DESCRIPTION
**Pull request by eitanlees**

### CODE REVIEW

File: altair/examples/line_chart_with_generator.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+"""
+# category: line charts
+
+import altair as alt
+
+source = alt.sequence(start=0, stop=12.7, step=0.1, as_='x')
+
+alt.Chart(source).mark_line().transform_calculate(
+    sin='sin(datum.x)'
+).transform_calculate(
```
Comment (jakevdp) [inline altair/examples/line_chart_with_generator.py]: You can put both ``sin`` and ``cos`` as arguments to a sinlge ``transform_calculate`` call.

**Comment by jakevdp** [comment]
This is great, thanks! One small change and we can get it merged.

File: altair/examples/line_chart_with_generator.py (hunk 1 above)
Comment (eitanlees) [reply altair/examples/line_chart_with_generator.py]: oh I didn't know that!

**Comment by eitanlees** [comment]
Changes made!

**Comment by eitanlees** [comment]
This PR should be good to go.

**Pull request merged**